### DISCUSSION TITLE
Like search does not work in Filter on Data Provider in Form

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#8762] [Help] [answered]
I've followed the instructions on how to build a search using a form and text input (flexible way), as per point 2 in https://docs.budibase.com/docs/adding-a-search-field

This works fine, but Like does not seem to work. I want to search a text field in an internal/BB table. Version 2.1.28

Does anybody know what I might overlook, and/or have a solution for this? Thanks in advance.

![Screenshot 2022-11-22 at 07 58 11](https://user-images.githubusercontent.com/4005405/203247085-d3335d91-b5be-42fb-8e48-7d7c0c61b7de.png)
![Screenshot 2022-11-22 at 07 58 38](https://user-images.githubusercontent.com/4005405/203247096-8ab60a4d-ead5-43a0-8f12-953fdcd696ef.png)

**Comment by ecxs** [comment]
Like does not work in Filter either ...
![Screenshot 2022-11-22 at 09 06 17](https://user-images.githubusercontent.com/4005405/203259078-463c349b-5180-47ef-94c1-c047ddcdf474.png)

**Comment by melohagan** [comment, marked as answer]
Hey @ecxs This is a known issue: https://github.com/Budibase/budibase/issues/7467

**Reply by ecxs** [reply]
Hi @melohagan thanks for letting me know. You may want to add this to your documentation at https://docs.budibase.com/docs/adding-a-search-field ... may safe some other new/first time users a lot of time ;-)
